glean-core/ffi/src/lib.rs

```diff
@@ -299,6 +300,116 @@ pub extern "C" fn glean_boolean_test_get_value(
     })
 }
 
+// *** Start of the Datetime FFI part ***
```

Comment:
Yeah, we really need to reorganize this file and split it up.

Reply:
Yes! We have a bug for  that already, see [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1553800).